mkdocs/docs/HPC/easybuild.md

```diff
@@ -141,8 +141,10 @@ make them accessible for loading:
 </code></pre>
 
 It makes sense to put this `module use` command and all `export`
-commands in your `.bashrc` login script. That way you don't have to type
+commands in your `.bashrc` login script.
+That way, you don't have to type
 these commands every time you want to use EasyBuild or you want to load
-modules generated with EasyBuild. See also [the section on `.bashrc` in
+modules generated with EasyBuild.
+See also [the section on `.bashrc` in
 the "Beyond the basics" chapter of the intro to
 Linux](https://hpcugent.github.io/vsc_user_docs/linux-tutorial/beyond_the_basics/#bashrc-login-script).
```

Comment:
We should avoid hardcoding the URL here, just refer to the `.md` file?

Please also check for other occurences of `hpcugent.github.io`, and get rid of those (in a separate PR)

Reply:
Fixed in #668